spec/requests/v0/my_va/submission_statuses_spec.rb

```diff
@@ -4,7 +4,7 @@
 require 'forms/submission_statuses/gateway'
 
 RSpec.describe 'V0::MyVA::SubmissionStatuses', type: :request do
-  let(:user) { build(:user, :loa3, :with_terms_of_use_agreement) }
```

Comment:
nit: we may also want to update the swagger_spec to reflect this change:

https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/requests/swagger_spec.rb#L3742